src/ui/NativeApp.vue

```diff
@@ -41,6 +41,9 @@ export default {
 </script>
 <style>
 body {
+  position: absolute;
+  inset: 0;
```

Comment:
This breaks the NewAccount.vue

Need to make sure that these fixes don't break the extension version